pkg/api/mlflow/dao/convertors/log.go

```diff
@@ -84,3 +76,14 @@ func ConvertLogBatchRequestToDBModel(
 	}
 	return metrics, params, tags, nil
 }
+
+func convertParam(runID, key string, valueInt *int64, valueFloat *float64, valueStr *string) *models.Param {
+	param := &models.Param{
+		Key:        key,
+		RunID:      runID,
+		ValueInt:   valueInt,
+		ValueFloat: valueFloat,
+		ValueStr:   valueStr,
+	}
+	return param
+}
```

Comment:
well i left it here in case we want to do some kind of inbound processing, eg convert string to float where that seems to be the intention. it's probably indifferent otherwise whether inline or function.

Reply:
public function delegating to private is pretty common, but I can remove it